scripts/populate_ingest/populate_ingest_from_nxn.py

```diff
@@ -0,0 +1,178 @@
+# --- core imports
+import argparse
+from datetime import datetime
+import itertools
+import logging
+import pprint
+import re
+import sys
+
+# --- third party library imports
+import Levenshtein
+
+# --- application imports
+from convert.conversion_utils import get_accessions
+from convert.europe_pmc import EuropePmcConverter
+from data_entities.nxn_db import NxnDatabase
+from services.europe_pmc import EuropePmc
+from services.nxn_db import NxnDatabaseService
+from services.ingest import QuickIngest
+
+
+# INGEST_URL = "https://api.ingest.archive.data.humancellatlas.org"
+INGEST_URL = "http://localhost:8080"
```

Comment:
read config values from a config.py file that in turn reads them from environment variables. They way switching envs and running in a container would be easy.

See https://12factor.net/config

Reply:
this was changed in the latest commit, which specifies the default ingest url as the prod url and can take a url for some other environment as a script parameter.

Is that alright or should we go down the config route? there's only one constant so using a config feels a like an overkill

Reply:
It is a good practice to have configs in a dedicated file with the possibility of overriding by environment variables. If you think a single configuration parameter does not constitute a config file, so be it. It can be created for the next configuration param.
Just make sure you allow overriding using environment variables.